gcloud/bigtable/column_family.py

```diff
@@ -15,6 +15,59 @@
 """User friendly container for Google Cloud Bigtable Column Family."""
 
 
+class GarbageCollectionRule(object):
+    """Table garbage collection rule.
+
+    Cells in the table fitting the rule will be deleted during
+    garbage collection.
+
+    .. note::
+
+        At most one of ``max_num_versions`` and ``max_age`` can be specified
+        at once.
```

Comment:
Let's just pull the `to_pb` methods into this PR when we split the classes:  they are going to be one-liners, and easier to test.